apps/web-portal-client/lib/features/auth/hooks/auth.jsx

```diff
@@ -19,5 +20,11 @@ export function useAuth() {
           dispatch(error(err))
         })
     },
+    logout: () => {
+      dispatch(logout())
```

Comment:
Do we have an API method to do logout?

Reply:
No, we have not. It is not clear for now what auth mechanism we will use. I can add some plug

Reply:
so extracting to separate task